### ISSUE TITLE
2.10.0 -  TraitError: The 'value' trait of a HTML instance must be a unicode string...

### ISSUE THREAD

**Issue by rynmccrmck** [pandas-profiling/pandas-profiling#668]
**Describe the bug**

Hi there - Looks like latest release (2.10.0) has broken a to_widgets functionality as outlined in the `Getting started` section of the docs. Confirmed eolling back to 2.9.0 does not produce the issue.

**To Reproduce**

```
# pandas_profiling==2.10.0
import numpy as np
import pandas as pd
from pandas_profiling import ProfileReport

df = pd.DataFrame(
    np.random.rand(100, 5),
    columns=["a", "b", "c", "d", "e"]
)

profile = ProfileReport(df, title='Pandas Profiling Report', explorative=True)

profile.to_widgets()

```
Returns:
```
TraitError: The 'value' trait of a HTML instance must be a unicode string, but a value of Numeric <class 'visions.types.type.VisionsBaseTypeMeta'> was specified.
```

 

**Version information:**
2.10.0

**Additional context**

**Comment by aidamanzano** [comment]
I have this same issue too, do you know if it has been fixed?

**Comment by rynmccrmck** [comment]
> I have this same issue too, do you know if it has been fixed?

@aidamanzano I don't think so yet, but you can downgrade to 2.9.0

**Comment by MarvinTeichmann** [comment]
I have the issue as well.

**Comment by ons-cherif** [comment]
Me too :(

**Comment by mukeshmithrakumar** [comment]
any updates on this issue?

**Comment by sbrugman** [comment]
Fixing it now, will release the fix together with some others soon.

**Comment by muthugit** [comment]
Version 2.9.0 works fine
```
pip install pandas-profiling==2.9.0
```

**Comment by mukeshmithrakumar** [comment]
Thanks for the quick fix fellows :)

**Comment by engrarqam** [comment]
Thanks a lot, Mukesh bro you solved my problem.

**Comment by engrarqam** [comment]
Thanks a lot, muthugit bro you solved my problem.

**Comment by luckrols** [comment]
Same issue here with version pandas-profiling version 2.10.0

**Comment by sbrugman** [comment]
Fix is available in develop, will release it tonight.

**Comment by sbrugman** [comment]
@rynmccrmck I've just released v2.10.1 that should resolve your issue!

(cc @aidamanzano @MarvinTeichmann @ons-cherif @analuizaypessoa @mukeshmithrakumar @muthugit @engrarqam @luckrols)

_Want me to continue to support this package? Please consider [sponsoring](https://github.com/sponsors/sbrugman)_

**Comment by mihaelagrigore** [comment]
Managed to solve this issue after I stopped using the pandas-profiling that gets installed with 
`conda install pandas-profiling`
And went instead for 
`pip install pandas-profiling`

I did not check which version conda had installed, sorry. But pip installed the latest (pandas-profiling-2.10.0). Maybe conda was getting the same version.

But for some reason now I no longer encounter the:
`TraitError: The 'value' trait of a HTML instance must be a unicode string, but a value of Numeric <class 'visions.types.type.VisionsBaseTypeMeta'> was specified.`
that I had before.
